### ISSUE TITLE
Add provider version for darwin_arm64

### ISSUE THREAD

**Issue by bertrandpgn** [hectorj/terraform-provider-googlesiteverification#19]
Hi,

I really like using this provider and I recently switched to a M1 mac which requires a darwin_arm64 provider version, that is not in the current assets of the v0.4.2 release. I am getting this error using `Terraform v1.0.10 on darwin_arm64`

```
|
│ Error: Incompatible provider version
│ 
│ Provider registry.terraform.io/hectorj/googlesiteverification v0.4.2 does
│ not have a package available for your current platform, darwin_arm64.
│
```

I think the correct variables for the go cross-compiling are GOOS=darwin and GOARCH=arm64, but yet they already are listed in the [.goreleaser.yml](https://github.com/hectorj/terraform-provider-googlesiteverification/blob/master/.goreleaser.yml#L25-L30) file. 

Could you consider adding this version ?

Let me know if I can help in any way, I'd be glad to.

**Comment by hectorj** [comment]
Fixed with a6004c0f7c7c833ac41255885347759fe3b958ee and published as v0.4.3

Tell me if it works (I don't have nor want a Mac to test it).

> I really like using this provider

Very happy to hear this!

**Comment by bertrandpgn** [comment]
Works great thanks !
